3rdparty/CMakeLists.txt

```diff
@@ -379,18 +379,11 @@ if (WIN32)
       IMPORTED_IMPLIB ${GLOG_ROOT}-build/glog${CMAKE_IMPORT_LIBRARY_SUFFIX})
   endif ()
 else ()
-  # TODO(andschwa): Build with CMake instead when glog is updated.
-  set(GLOG_CONFIG_CMD  ${GLOG_ROOT}/src/../configure --with-pic GTEST_CONFIG=no --prefix=${GLOG_ROOT}-build)
-  if (FREEBSD)
-    list(APPEND GLOG_CONFIG_CMD LDFLAGS=-lexecinfo)
-  endif ()
-  set(GLOG_BUILD_CMD   ${MAKE_PROGRAM})
-  set(GLOG_INSTALL_CMD ${MAKE_PROGRAM} install)
-
   set_target_properties(
     glog PROPERTIES
-    IMPORTED_LOCATION ${GLOG_ROOT}-build/lib/libglog${LIBRARY_SUFFIX}
-    INTERFACE_INCLUDE_DIRECTORIES ${GLOG_ROOT}-build/include)
+    INTERFACE_INCLUDE_DIRECTORIES "${GLOG_ROOT}-build;${GLOG_ROOT}/src"
```

Comment:
As far as I understand, the `${GLOG_ROOT}-build/include` was the artefact produced by `make install`.

After I've switched our cmake build to building glog with cmake, th einstall step disappeared. Probably the right solution will be removing `INSTALL_COMMAND   ${CMAKE_NOOP}` and setting the variables for cmake install to work properly - I'll try this.

Simply removing `INSTALL_COMMAND` does not work (apparently, because glog's CMakeLists do not use the `INSTALL_DIR` filled by the ExternalProject).

Reply:
Tried this. To do this properly, I need a good understanding of what is going on inside the Windows build, as well as an ability to debug the Windows build.